lib/injectHash.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const Logr = require('logr');
+const glob = require('glob');
+const fs = require('fs');
+const debounce = require('lodash.debounce');
+
+const log = new Logr({
+  type: 'cli',
+  renderOptions: {
+    cli: {
+      lineColor: 'yellow'
+    }
+  }
+});
+
+const cache = {};
+
+const writeFile = debounce((file, contentString) => {
+  fs.writeFileSync(file, contentString);
```

Comment:
now that I think about it, we are already deboucing higher up the funnel, so we might not need this